Expensify/App
# Enable Hermes JS Engine on Android

Hermes JS Engine is supposed to make our Android app smaller and faster, this PR enables it!

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/142237

### Tests
Started up the Android app and clicked around a bunch. Opened and closed the app a few times. Checked out the size difference of the app (About 20% smaller with Hermes). Tested the debugger. Everything seemed to work great!

I also updated the readMe to explain the new debugging. 

I'm also going to send the following email:
```
[Some Action Required] Upgrading Expensify.Cash to use Hermes!

Hello Engineering Friends :wave: 

If you're working on Expensify.cash, this email is for you! Due to [this PR](PR) we are now using the Hermes JS engine for the Android app which should improve the speed, and decrease the size of the app. 

All you need to do to set it up is: 
1. Run `cd android && ./gradlew clean` from `Expensidev/ReactNativeChat` to remove the old version of the app
2. Run `npm run android` to make the new version with Hermes enabled.

The only change you should really notice is when debugging as the process has changed slightly. I've updated the [ReadMe](link) to explain the new process, but it has been fairly painless in my experience.

If you have any questions take a look at the [Hermes docs ](https://reactnative.dev/docs/hermes) or shoot me a message on E.cash!

Cheers,
Brandon
```
